libs/ui/src/themes/render_with_themes.tsx

```diff
@@ -110,6 +110,14 @@ export interface VxQueryFunctions {
     buttonText: string | RegExp,
     opts?: ButtonQueryOptions
   ): ReturnType<queries.FindByRole>;
+  queryButton(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.QueryByRole>;
+  queryAllButtons(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.AllByRole>;
```

Comment:
We have all the others (get, getAll, find, findAll) so we might as well have all 6.